sources/world/GeoScribbles.geojson

```diff
@@ -0,0 +1,29 @@
+{
+    "geometry": null,
+    "properties": {
+        "attribution": {
+            "required": false,
+            "text": "GeoScribble contributors",
+            "url": "https://wiki.openstreetmap.org/wiki/GeoScribble"
+        },
+        "description": "Geometric survey notes, aged three weeks at most",
+        "i18n": false,
+        "id": "geoscribble-latest",
+        "available_projections": [
+            "EPSG:3857",
+            "EPSG:4326"
+        ],
+        "max_zoom": 22,
+        "name": "GeoScribble latest notes",
```

Comment:
```suggestion
        "name": "GeoScribble latest notes (Every Door)",
```

Reply:
I would not tie this to a single editor. Pretty sure we'll see this layer being used from other apps before the year ends.

Reply:
Agreed, that might very well be. But it will be easy to update it here, once that happens. Right now, I don't see a way that a user would be able to understand where those GeoScribbles came from just by looking at the title and description in the iD Editor Sidebar. — Adding a bit more context helps a lot.